src/pages/vault/tutorials/index.tsx

```diff
@@ -1,15 +1,15 @@
 import productData from 'data/vault.json'
 import { LearnProductData } from 'types/products'
 import {
-  getProductTutorialsPageProps,
-  ProductTutorialsPageProps,
+  getProductTutorialsViewProps,
+  ProductTutorialsViewProps,
```

Comment:
Have renamed this in an effort to more cleanly differentiate between `ProductPageBlock`, the "block" type we receive from the learn client, and `ProductViewBlock`, the "block" type we pass to components in the front end. This is definitely a bit of a half-measure (I feel like the names are maybe just _slightly_ more differentiated), so very open to input here 🙇

Reply:
Works for me!